symbulate/plot.py

```diff
@@ -40,4 +41,108 @@ def plot(*args, **kwargs):
     except:
         plt.plot(*args, **kwargs)
         
+def is_discrete(heights):
+    return sum([(i > 1) for i in heights]) > .8 * len(heights)
+
+def count_var(x):
+    counts = {}
+    for val in x:
+        if val in counts:
+            counts[val] += 1
+        else:
+            counts[val] = 1
+    return counts
+    
+def compute_density(values):
+	density = gaussian_kde(values)
+	density.covariance_factor = lambda: 0.25
+	density._compute_covariance()
+	return density
+
+def setup_ticks(pos, lab, ax, axis):
```

Comment:
Suggestion: I believe that you can set ticks alternatively using `ax.xaxis.set_ticks` and `ax.yaxis.set_ticks`. Then you wouldn't have to pass in both `ax` and `axis`. You can just pass around either `ax.xaxis` or `ax.yaxis`, which would specify which axis you want to modify, and get rid of the `axis=` argument. Can you explore this to see if it would work?